e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/report-reconciliation/add-payment.spec.js

```diff
@@ -155,7 +155,7 @@ context('PDC_RECONCILE users can add a payment to a report', () => {
 
     cy.contains('Premium payments').should('exist');
 
-    cy.assertText(premiumPaymentsTable.status(FEE_RECORD_ID_ONE), FEE_RECORD_STATUS.MATCH);
+    cy.assertText(premiumPaymentsTable.status(FEE_RECORD_ID_ONE), 'Match');
```

Comment:
should this not be from a constant?

Reply:
That'd be great - there is some confusion around these constants IMO. `FEE_RECORD_STATUS.MATCH` is in uppercase. The actual status is not.